src/components/container/LabResultsSummary/LabResultsSummary.tsx

```diff
@@ -87,6 +88,7 @@ export default function (props: LabResultsSummaryProps) {
     }
 
     return <Action
+        innerRef={props.innerRef}
```

Comment:
Line `78` may need the `ref`.

Reply:
Good catch on all of these.  Did end up having to wrap most of these in a <div> since the ref can't be both an HTMLDivElement ref and HTMLButtonElement ref - but that seems to work fine.